include/mxnet/base.h

```diff
@@ -532,6 +532,9 @@ inline std::ostream& operator<<(std::ostream &out, const Context &ctx) {
   return out;
 }
 
+// convert nnvm symbol to a nnvm graph.
+nnvm::Graph Symbol2Graph(const nnvm::Symbol &s);
```

Comment:
@waytrue17 is this needed now?

Reply:
right.. I'll remove it in the next commit

Reply:
still planning to remove this?

Reply:
yes

Reply:
removed

Reply:
Yes, will remove it within the next commit